### PR TITLE
tasks 61-70

### PR DESCRIPTION
**Pull request by maycatsby**

### CODE REVIEW

File: tasks/64_sum-odd-fibonacci/sum-odd-fibonacci.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  let result = 1;
+  let fibonacci = [0, 1];
+  let curFibNum = 0;
+  for (let i = 2; curFibNum <= num; i++) {
+    if (curFibNum % 2) {
+      result += curFibNum;
+    }
+    curFibNum = fibonacci[i - 2] + fibonacci[i - 1];
+    fibonacci.push(curFibNum);
+  }
```
Comment (asmyk) [inline tasks/64_sum-odd-fibonacci/sum-odd-fibonacci.js:11]: ooo nice solution ;)

File: tasks/66_sum-primes/sum-primes.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    return 0;
+  }
+  let sumOfPrimes = 2;
+  let prime;
+  for (let j = 3; j <= num; j += 2) {
+    prime = true;
+    for (let i = 2; i <= parseInt(Math.sqrt(j)); i++) {
+      if (j % i === 0) {
+        prime = false;
+        break;
```
Comment (asmyk) [inline tasks/66_sum-primes/sum-primes.js:12]: is break necessary here?

File: tasks/68_drop-elements/drop-elements.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+export const dropElements = (arr, func) => {
+  let solution = [];
+  for (let i = 0; i < arr.length; i++) {
+    if (func(arr[i])) {
+      solution = arr.slice(i);
+      break;
```
Comment (asmyk) [inline tasks/68_drop-elements/drop-elements.js:6]: break?

File: tasks/69_unpack-elements/unpack-elements.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+export const unpackArray = (arr) => {
+  return arr.reduce((Arr, val) => {
+    return Array.isArray(val) ? Arr.concat(unpackArray(val)) : Arr.concat(val);
```
Comment (asmyk) [inline tasks/69_unpack-elements/unpack-elements.js:3]: nice :)

**Review by asmyk — APPROVED** [review]

File: tasks/65_smaller-common-multiple/smaller-common-multiple.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+export const smallestCommons = (arr) => {
+  const arrSort = arr.sort((a, b) => a - b);
+  for (let i = 1; ; i++ ) {
+    let number = arrSort[1] * i;
+    let isDivided = 1;
```
Comment (tsvetkovv) [inline tasks/65_smaller-common-multiple/smaller-common-multiple.js:5]: why not boolean?

**Review by tkachukivan — APPROVED** [review]

**Review by tsvetkovv — APPROVED** [review]

**Pull request merged**